src/openrct2/windows/top_toolbar.c

```diff
@@ -781,6 +790,11 @@ static void window_top_toolbar_invalidate(rct_window *w)
 	else
 		w->pressed_widgets &= ~(1 << WIDX_PAUSE);
 
+	if (!gGameSoundsOff)
+		window_top_toolbar_widgets[WIDX_MUTE].image = 0x20000000 | SPR_G2_TOOLBAR_MUTE;
```

Comment:
It would be better to use a constant, rather than the magic number `0x20000000`.